src/third/redux-persist/createPersistor.js

```diff
@@ -86,12 +87,10 @@ export default function createPersistor (store, config) {
       stateIterator(incoming, (subState, key) => {
         try {
           let data = deserializer(subState)
-          let value = transforms.reduceRight((interState, transformer) => {
-            return transformer.out(interState, key)
-          }, data)
+          let value = data
           state = stateSetter(state, key, value)
         } catch (err) {
-          if (process.env.NODE_ENV !== 'production') console.warn(`Error rehydrating data for key "${key}"`, subState, err)
+          logging.warn('Error rehydrating data for a key', { key, subState, err })
```

Comment:
`subState` here looks like potentially a lot of data. Maybe just drop it?

Alternatively could try to include a prefix or something, to help identify whether it was compressed (and labeled with our compression header.) But I'm not sure that will necessarily add anything vs. the error object.